code/game/atoms.dm

```diff
@@ -290,6 +290,7 @@
 				to_chat(user, "[total_volume] units of various reagents")
 		else
 			to_chat(user, "Nothing.")
+	SendSignal(COMSIG_PARENT_EXAMINE, user)

```

Comment:
Either really. I was mostly thinking that examine order should be constant and controlled and afaik signal order would depend strictly on types of components here ?